settlement-distributions/bid-distribution/README.md

```diff
@@ -0,0 +1,31 @@
+# Engine to generate protected event Settlements for Bidding
```

Comment:
Flagging a place with "protected events" present, not sure whether there are some places where it's still legit or not (whether the term is fully replaced by the new one or it's still used to refer to multiple types of events).

Reply:
This was meant to be legit. I use the PSR and "protected event" interchangeably for event of slashing validator.
The protected event is for bidding or for institutional validators. That's how it is in my mind but terminology could be more clear, I understand. But I'm not planning to force it right now.